### DISCUSSION TITLE
issue about k8s metric

### DISCUSSION THREAD

**Discussion by ufoac** [SigNoz/signoz#2439] [General]
I've been trying to expose metrics from k8s to SigNoz,  strictly followed by https://signoz.io/docs/tutorial/kubernetes-infra-metrics/#list-of-metrics

Everything goes well, except some types of container/pod metrics are missing, such as **container_memory_usage**, **container_cpu_utilization**, while **k8s_container_memory_request** **k8s_container_cpu_limit** **system_xx** ... are correctly observed in the dashboard.

This has bothered me for a while. Any help would be appreciated.

**Comment by srikanthccv** [comment]
Continuing here https://github.com/SigNoz/signoz/issues/2440; it will be easier to track progress there if it is indeed a bug with our charts.

**Comment by Nageshbansal** [comment]
Hey @ufoac, closing this. Please follow up on #2440.
